docs/UserGuide.md

```diff
@@ -11,18 +11,18 @@
    > Having any Java 8 version is not enough. <br>
    This app will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `lifekeeper.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for your Lifekeeper.
 3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
+   > <img src="images/UIprototype.png" width="600">
```

Comment:
about the UI mock up:

is the right pane only for legend? if so, you are wasting some real precious real-estate on the GUI. try using that to provide some useful information to the user. your legend can be just a small part at the bottom (or can be removed totally by mentioning to the user about various colors in the user guide)